_includes/styles.css

```diff
@@ -1,74 +1,324 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 :root {
-  /* setting color scheme for light/dark text */
   color-scheme: light dark;
-  --light-color: #ffcfb8;
-  --dark-color: #113134;
+  --md-primary: #ea6a00;
+  --md-primary-light: #ff8a33;
+  --md-primary-dark: #b85200;
+  --md-surface: #ffffff;
+  --md-surface-container: #f3edf7;
+  --md-surface-container-high: #ece6f0;
+  --md-on-surface: #1c1b1f;
+  --md-on-surface-variant: #49454f;
+  --md-outline: #79747e;
+  --md-background: #fef7ff;
+  --md-on-primary: #ffffff;
 }
 
 html {
-  --primary-color: var(--dark-color);
-  --secondary-color: var(--light-color);
+  font-family: -apple-system, system-ui, "Helvetica Neue", Helvetica, Arial, sans-serif;
+  background: var(--md-background);
+  min-height: 100vh;
+  transition: background 0.3s ease;
+}
 
-  background-color: var(--primary-color);
-  text-align: center;
+body {
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
+  overscroll-behavior: none;
+}
+
+.theme-picker {
   position: fixed;
-  width: 100%; 
-  height: 100%;
-  margin: 0;
-  padding: 0;
-  /* system specific font */
-  font-family: -apple-system, system-ui, "Helvetica Neue", "Helvetica", "Arial",
-  sans-serif;
+  top: 20px;
+  right: 20px;
+  background: var(--md-surface-container-high);
```

Comment:
surface container higher so it's more visible https://m3.material.io/blog/tone-based-surface-color-m3